docs/rsdl/rapid-rsdl-capabilities.md

```diff
@@ -0,0 +1,496 @@
+---
+id: rsdl-capabilities
+title: RSDL Capabilities
+---
+
+# Expressing Capabilities in RAPID Schema Definition Language (RSDL)
+
+RAPID uses a Schema Definition Language ([RSDL](./rsdl-intro)) to describe the structure of a REST service. Developers can be annotate their schema model with [capabilities](./rsdl-abnf#model-capabilities) in order to describe the capabilities of their service.
+
+- [Annotating Model Elements](#annotating-model-elements)
+  - [Reading Data](#support-for-reading-data)
+    - [Read Options](#read-options)
+      - [expand Option](#expand-option)
+  - [Listing Collections](#listing-collections)
+    - [List Options](#list-options)
+      - [filter Option](#filter-option)
+      - [orderby Option](#orderby-option)
+      - [top/skip Options](#top-and-skip-options)
+      - [count Option](#count-option)
+  - [Modifying Data](#support-for-modifying-data)
+    - [Creating Resources](#create-support)
+      - [Create options](#create-options)
+    - [Updating Resources](#update-support)
+      - [Update options](#update-options)
+    - [Replacing Resources](#replace-support)
+      - [Replace Options](#replace-options)
+    - [Deleting Resources](#delete-support)
+      - [Delete Options](#delete-options)
+  - [Invoking Operations](#operation-support)
+  - [Default Capabilities](#default-capabilities)
+- [Annotating Paths](#annotating-paths)
+
+# Annotating Model Elements
+
+Model elements are the service, operations, types, and properties defined in in an RSDL Schema.
+
+Capabilities applied to model elements determine the set of paths and capabilities implemented by the service. The developer can generate a set of paths and capabilities from the annotated model to explore or further refine the supported functionality.
+
+Capabilities for a model element are specified in curly braces following the model element definition. Omitting a capability within the curly braces means that capability is not supported. If no curly braces follow the model element, a [default set of capabilities](#default-capabilities) is assumed.
+
+## Support for reading data
+
+A top level singleton or single-valued reference property can be annotated with the `READ` capability to show that it supports reading.
```

Comment:
Why introduce "reference property"? What exactly is it, compared to a navigation property?